web/src/components/Elements/Buttons/TextButton/TextButton.css

```diff
@@ -0,0 +1,15 @@
+.text-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: transparent;
+    border: none;
```

Comment:
Gereksiz sanki?

Reply:
5-6 olmazsa styling bozulacağına eminim. 3-4 için bakacağım